OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -51,7 +51,24 @@ The parent class, Muscle.h, provides
 
 namespace OpenSim {
 
+//==============================================================================
+//                   Millard2012AccelerationMuscle Exceptions
+//==============================================================================
+class Millard2012AccelerationMuscleInvalidFlag : public Exception {
```

Comment:
A more logical design is for the internal method to return an enum so it is impossible for the caller to receive an unexpected flag. From your response, it seems this is considered to be an important piece of code, so I will implement the redesign and remove the MuscleInvalidFlag exceptions.